packages/components/select/src/hidden-select.tsx

```diff
@@ -99,19 +99,16 @@ export function useHiddenSelect<T>(
       ["data-a11y-ignore"]: "aria-hidden-focus",
     },
     inputProps: {
```

Comment:
Could probably delete the hidden input element, see https://github.com/adobe/react-spectrum/pull/7200